### ISSUE TITLE
Vignette for comparison with altnerative solutions

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/modelbased#108]
@IndrajeetPatil @bwiernik since you're arguably the most knowledgeable of other alternatives; what say you about creating a vignette to compare modelbased to other alternatives (broom::augment; tidymodels; ggeffects; raw emmeans; ...) with the threefold goal of 1) showing how to do something with other packages; 2) eventually help users transitioning; 3) get an overview of areas of improvement where modelbased lacks some features or is more complicated to use?

Could you throw at me some examples of problems that you actually had to address with them / features that you use / things that you like with alternatives, so that I can come up with the modelbased alternative and put it all in a vignette? Also pinging @strengejacke @mattansb  for ggeffects

**Comment by strengejacke** [comment]
Makes sense. I must dig a bit deeper into the latest modebased API, though.

**Comment by DominiqueMakowski** [comment]
How can we kickstart that? should we all throw in some problems? and then we solve them using the different packages? Or we all throw some feature using one package and we try replicating results with the others?

**Comment by DominiqueMakowski** [comment]
Should we do that in the https://github.com/easystats/modelbased/discussions so we can then comment on others replies for better / alternative / more concise solutions (within the same package)? and then assemble the replies into a vignette?

**Comment by mattansb** [comment]
(I would love a ggeffects (❤) vs modelbased vignette...)

**Comment by IndrajeetPatil** [comment]
Similar to https://github.com/easystats/easystats/issues/104

Not sure if this vignette should reside here, or in `easystats`.

**Comment by bwiernik** [comment]
I think easy vs ggeffects and easy vs broom should be separate vignettes residing in easystats

**Comment by DominiqueMakowski** [comment]
these type of comparison vignettes could be good first contributions for new contributors as ways of getting more familiar with easystats + github + R, if you have any potentially interested students or interns this summer or after
